addons/hr_expense/tests/test_expenses.py

```diff
@@ -1,6 +1,6 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 import base64
-
+from datetime import date
```

Comment:
```suggestion
from datetime import date

```
According to PEP8
Imports should be grouped in the following order:

Standard library imports. (base64, datetime)
Related third party imports. (freezegun)
Local application/library specific imports. (odoo)

(Also `__future__` imports are always the first ones)
I prefer to keep those groups separated by a blank line

Note that here you could access date objects through `odoo.fields.Date(?=time).to_date(?=time)(isostring)`
Or just put a iso string e.g. `'2021-01-01'` (related to your create dicts)
Not sure you need this :)